DeviceSettings/res/values/aosip_strings.xml

```diff
@@ -28,7 +28,7 @@
     <string name="panel_category_summary">Modes for panel color correction</string>
     <string name="off_mode_title">Off</string>
     <string name="tile_panel_mode">Panel modes</string>
-    <string name="dc_mode_summary"></string>
+    <string name="dc_mode_summary">Reduces eye strain in low light conditions but may result in reduced image quality and poor color production.</string>
```

Comment:
```suggestion
    <string name="dc_mode_summary">Useful for people who are sensitive to PWM\nSide effects include eye strain, nausea and migraines.\nEnabling this implies less accurate display colors</string>
```
As it's not a fact it reduces these for everyone, only a small portion are really sensitive to this.
Also it necessarily causes worse colors (so not may) - but it should not affect "quality".